software/Test_Robot_Software/video_stream.py

```diff
@@ -0,0 +1,62 @@
+import image_processor
+import camera
+import cv2
+import time
+
+def main():
+    debug = True   
+    
+    cam = camera.RealsenseCamera(exposure = 100)
+    
+    processor = image_processor.ImageProcessor(cam, debug=debug)
+
+    processor.start()    
+
+    start = time.time() 
+    fps = 0
+    frame = 0
+    frame_cnt = 0
+    try:
+        while True:
+            
+            # has argument aligned_depth that enables depth frame to color frame alignment. Costs performance
+            processedData = processor.process_frame(aligned_depth=False)
+            largest = max(processedData.balls, key = lambda ball: ball.size, default=None)
```

Comment:
The processed data ball list is already sorted

Reply:
can I just use `largest = processed_data.balls[0]` ?